recipes/flecs/all/patches/less-warnings.patch

```diff
@@ -0,0 +1,32 @@
+--- a/cmake/target_default_compile_warnings.cmake
```

Comment:
It should be removed, it doesn't patch the building.

Reply:
> 
> 
> The commit/patch name is misleading. It removes invalid compiler warnings...